search-sentiments-map/public/scripts/search-topic-script.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+/**
+ * Updates the current search topic for the given country and search topic.
+ */
+function searchTopic() {
+  let topic = document.getElementById("search-topic").value;
+  let topTrends = getTopTrends();
+  // Maximum of 3 countries. If given more, only take first 3.
+  let countries = $('#country-select').val().slice(0, 3);
```

Comment:
Create a constant for 3. MAX_SELECTED_COUNTRIES maybe.
So this can be updated in the future if needed.